packages/doc-site/stories/components/gauge/Docs.mdx

```diff
@@ -0,0 +1,177 @@
+import { Canvas, Meta, Story } from '@storybook/blocks';
+
+import * as GaugeStories from './Gauge.stories';
+
+# Gauge
+
+The Key Performance Indicator (Gauge) component provides a compact representation when you need an overview of your asset properties. This overview gives you the most critical insights into the overall performance of your devices, equipment, or processes. With the Gauge, you can interact with IoT data from one or more data sources.
```

Comment:
Gauge is not a Key Performance Indicator. We don't want to confuse our customers! Should just be " The Gauge component provides ..."

Reply:
Done